### DISCUSSION TITLE
Insert shape and floating text in reveal.js

### DISCUSSION THREAD

**Discussion by Lightup1** [quarto-dev/quarto-cli#2608] [Q&A] [answered]
### Is it possible to insert shape and floating text in reveal.js by quarto as in powerpoint?

For example, to insert arrow line and floating text box like this 
![image](https://user-images.githubusercontent.com/32999017/192187806-acb43797-9cfd-41d8-b647-c44f9b04ebe7.png)
https://stackoverflow.com/questions/47756349/arrows-to-connect-annotational-boxes-with-content

**Comment by mcanouil** [comment, marked as answer]
Having floating boxes is "easy" (but will require tries and errors to properly positioned them) with HTML divs and css using the `position: absolute` propriety.
Drawing curve arrows in HTML is way harder especially if those are supposed to "connect" divs.
